services/self-service/src/test/java/com/epam/dlab/backendapi/resources/ProjectResourceTest.java

```diff
@@ -98,4 +115,11 @@ public void generateKeysWithException() {
         verify(keyService).generateKeys(getUserInfo());
         verifyNoMoreInteractions(keyService);
     }
-}
\ No newline at end of file
+
+    private ProjectActionFormDTO getProjectAvtionDTO() {
+        List<String> endPoints = new ArrayList<>();
+        endPoints.add("https://localhost:8083/");
+        String projectName = "DLAB";
+        return new ProjectActionFormDTO(projectName, endPoints);
```

Comment:
Use Collections.singletonList insead of ** endPoints**. And inline ProjectActionFormDTO object creation

Reply:
the singleton list has been used to reduce the code lines number